sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/client.py

```diff
@@ -762,6 +794,11 @@ def __init__(self, hostname, source, **kwargs):
         self._max_message_size = kwargs.pop("max_message_size", MAX_FRAME_SIZE_BYTES)
         self._link_properties = kwargs.pop("link_properties", None)
         self._link_credit = kwargs.pop("link_credit", 300)
+        self._timeout = kwargs.pop("timeout", 0)
+        self._timeout_reached = False
+        self._last_activity_stamp = time.time()
+        self._running_iter = False
+        self.auto_complete = kwargs.pop("auto_complete", True)
```

Comment:
We shouldn't need to add auto-complete do we?
I think that's a bit of uamqp hangover....